src/aerospike_vector_search/shared/admin_helpers.py

```diff
@@ -83,6 +83,50 @@ def _prepare_index_create(
         index_create_request = index_pb2.IndexCreateRequest(definition=index_definition)
         return (index_stub, index_create_request, kwargs)
 
+    def _prepare_index_update(
+            self,
+            namespace: str,
+            name: str,
+            index_labels: Optional[Dict[str, str]],
+            hnsw_update_params: Optional[types.HnswIndexUpdate],
+            timeout: Optional[int],
+            logger: logging.Logger
+    ) -> Tuple[index_pb2_grpc.IndexServiceStub, index_pb2.IndexUpdateRequest, Dict[str, Any]]:
```

Comment:
Similarly you should be able to use something like...
```suggestion
    ) -> tuple[index_pb2_grpc.IndexServiceStub, index_pb2.IndexUpdateRequest, dict[str, Any]]:
```
instead of importing the tuple type